cmd/cstor-pool-mgmt/controller/replica-controller/handler.go

```diff
@@ -280,6 +280,11 @@ func (c *CStorVolumeReplicaController) cVRAddEventHandler(
 				"CStorVolumeReplica %v is already imported",
 				string(cVR.ObjectMeta.UID),
 			)
+			if err := volumereplica.UpdateReplicaID(cVR); err != nil {
```

Comment:
why do we need `Update` in AddEventHandler.. if these changes are to take care of upgrade case, updateReplicaID need to be done only once, and its better to be done when upgrade of volume is triggered.. If I'm not wrong, upgrade is taken care by upgrade framework

Reply:
this is the upgrade PR related to this @mynktl https://github.com/openebs/maya/pull/1448 .. can you as well review that PR?